airsonic-main/src/main/webapp/WEB-INF/jsp/helpToolTip.jsp

```diff
@@ -10,26 +10,26 @@ PARAMETERS
 
 <script>
 $( function() {
-	$( document ).tooltip();
+  $( document ).tooltip();
 });
 </script>
 
 <style>
-	div.ui-tooltip {
-		max-width: 400px;
-	}
+  div.ui-tooltip {
+    max-width: 400px;
+  }
```

Comment:
These indents should be standard 4-space not 2.

Reply:
Apologies for this.  I didn't see that the rest of the file is already 2 space indent.  Probably better to keep it consistent rather than just change these few lines.